client/ayon_houdini/plugins/create/create_usd.py

```diff
@@ -96,7 +96,10 @@ class CreateUSDModel(CreateUSD):
     additional_parameters = {
         # Set the 'default prim' by default to the folder name being
         # published to
-        "defaultprim": '/`strsplit(chs("folderPath"), "/", -1)`',
+        "defaultprim": (
+            '/`ifs(strmatch("[0-9]", strsplit(chs("folderPath"), "/", -1)[0])'
+            ', "_", "")``strsplit(chs("folderPath"), "/", -1)`'
```

Comment:
With the complexity of this expression I start having the urge to make this a constant somewhere so it's maintained in one location. :)